### PR TITLE
Update Get-Process.md

### PR DESCRIPTION
**Pull request by doctordns**
Fixed name parameter details to show it accepts wildcards in the name

Version(s) of document impacted
------------------------------
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Management/Get-Process.md](https://github.com/doctordns/PowerShell-Docs/blob/patch-3/reference/5.1/Microsoft.PowerShell.Management/Get-Process.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C27%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711270943348088-1890%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=t6VGQofmAyF1gmP3ZaRTByoHxQHN58%2F5sikHlaTOQrU%3D&st=2017-11-27T10%3A05%3A32Z&se=2017-12-28T10%3A10%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by doctordns** [comment]
Thanks for the pull.

Out of curiosity, when is this change likely to hit the downloadable help files. The reason for asking this: I am teaching PowerShell this week and we noticed the error (fixed in my PR). Fixing it made for a fantastic demo of the new open source approach - I'd love my students to see the change, both online and in the downloadable help.

**Comment by sdwheeler** [comment]
@joeyaiello - Can you answer @doctordns question?

File: reference/5.1/Microsoft.PowerShell.Management/Get-Process.md [hunk 1]
```diff
@@ -355,7 +355,7 @@ Required: False
 Position: 0
 Default value: None
 Accept pipeline input: True (ByPropertyName)
-Accept wildcard characters: False
```
Comment (LGFN) [inline reference/5.1/Microsoft.PowerShell.Management/Get-Process.md]: y

**Pull request merged**